modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2555,13 +2554,22 @@ private void _executeObjectActions(
 	}
 
 	private void _fillDefaultValue(
-		long objectDefinitionId, Map<String, Serializable> values) {
+		String defaultLanguageId, long objectDefinitionId,
+		Map<String, Serializable> values) {
 
 		for (ObjectField objectField :
 				_objectFieldPersistence.findByObjectDefinitionId(
 					objectDefinitionId)) {
 
-			if (values.get(objectField.getName()) != null) {
+			Map<String, Object> localizedValues =
```

Comment:
Hey @carolmariaabb ,

I had to revert to the old implementation because the tests are failing due to the usage of Collections.singletonMap in multiple tests.